Automattic/wp-calypso
# VideoPress Onboarding v2

#### Proposed Changes

* Creates a new onboarding flow for VideoPress.
* See also pxWta-1z5-p2
* This is a culmination of many different PRs that were merged to this branch.

<img width="1066" alt="Screen Shot 2022-10-12 at 5 30 42 PM" src="https://user-images.githubusercontent.com/789137/195466344-588a09d7-6005-43db-ae6f-e1d77b3e8f98.png">

#### Testing Instructions



* Start the flow at http://calypso.localhost:3000/setup/intro?flow=videopress
* Walk through the onboarding, choosing a premium or better plan.
* You should end up with a Videomaker site.


Related to #
